packages/taquito-rpc/src/types.ts

```diff
@@ -1273,11 +1273,24 @@ export interface TezosGenericOperationError {
   id: string;
 }
 
+export interface TicketUpdates {
+  ticket_token: {
+    ticketer: string;
+    content_type: MichelsonV1Expression;
+    content: MichelsonV1Expression;
+  };
+  updates: {
+    account: string;
+    amount: string;
+  }[]
+}
+
 export interface OperationResultTransaction {
   status: OperationResultStatusEnum;
   storage?: MichelsonV1Expression;
   big_map_diff?: ContractBigMapDiff;
   balance_updates?: OperationBalanceUpdates;
+  ticket_updates?: TicketUpdates[]
```

Comment:
Related to the issue discussed during the Proto-ecosystem (nov 15), this property is named `ticket_updates` or `ticket_receipt`:

- contents -> metadata -> operation_result -> ticket_updates
- contents -> metadata -> internal_operation_result -> [ ] ->  ticket_receipt

Both cases are present in this block: https://limanet.ecadinfra.com/chains/main/blocks/10150/

We need to decide how to handle this. From what I understand, `ticket_receipt` might be replaced by `ticket_updates` in the next protocol... Should we go ahead and override  `ticket_receipt` for `ticket_updates`?

And make sure we have unit tests for both cases.

Reply:
added for the mean time ticket_receipt as the same type. so either could be accessed

we can remove the one that is removed when its done :D